src/main/webapp/about/dev-platform.jsp

```diff
@@ -0,0 +1,113 @@
+<%@ taglib uri="http://www.opensymphony.com/oscache" prefix="cache" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib uri="http://igniterealtime.org/website/tags" prefix="ir" %>
+<html>
+<head>
+    <title>a real time collaboration community site</title>
+    <meta name="body-id" content="home" />
+    <meta name="panel-name" content="home" />
+    <style media="screen">
+        @import "../styles/interior.css";
+    </style>
+</head>
+<body>
+
+<section id="ignite_body">
+
+    <main id="ignite_body_leftcol">
+
+        <article id="ignite_int_body">
+
+            <header id="ignite_body_header_with_content">
+                <h1>Open Realtime <span class="subtitle">Our Projects as Development platforms</span></h1>
+
+                <p>
+                    Discover how you can use our products as a platform to launch your idea!
+                </p>
+
+            </header>
+
+            <div class="ignite_int_body_padding">
+
+                <h2 class="larger">XMPP</h2>
+
+                <img src="../images/XMPP_logo.svg" alt="XMPP logo" align="right" style="margin-left: 10px; width: 200px; height: 200px"/>
+
+                <p>
+                    At the core of most of our projects lies XMPP. XMPP is open, mature, flexible and extensible, making it the protocol of choice for real-time communications over the Internet. It enables the reliable transport of structured data between individuals or applications. The <a href="https://xmpp.org">XMPP Standards Foundation</a> manages the development of the XMPP protocols.
+                </p>
+
+                <p>
+                    Numerous mission-critical business applications use XMPP, including chat and IM, network management and financial trading - and so could yours. With inherent security features and support for cross-domain server federation, XMPP is more than able to meet the needs of the most demanding environments.
+                </p>
+
+                <p>
+                    You can find more information on XMPP in our whitepaper called <a href="../support/articles/whitepaper_xmpp.jsp">"XMPP: The Protocol for Open, Extensible Instant Messaging"</a> or at the website of the <a href="https://xmpp.org">XMPP Standards Foundation</a>.
+                </p>
+
+                <h2 class="larger">Client-oriented Development</h2>
+
+                <p>
+                    <strong><a href="./projects/smack/">Smack</a></strong> is an Open Source XMPP client library for instant messaging and presence. A library for Android and Java, it can be embedded into your applications to create anything from a full XMPP client to simple XMPP integrations such as sending notification messages and presence-enabling devices.
```

Comment:
```suggestion
                    <strong><a href="./projects/smack/">Smack</a></strong> is an ppen source XMPP client library for instant messaging and presence. A library for Android and Java, it can be embedded into your applications to create anything from a full XMPP client to simple XMPP integrations such as sending notification messages and presence-enabling devices.
```

Reply:
I'll leave at capitalized for now, for consistency with other pages. I've always thought it to be a bit weird, but if we change that, we should do so everywhere.